### DISCUSSION TITLE
Réutilisation des modes de paiement

### DISCUSSION THREAD

**Discussion by cubse2** [PayGreen/documentation#20] [Q&A] [answered]
Bonjour,

Je dois mettre en place un système d'abonnement avec un paiement récurrent toutes les semaines.
Es-ce qu'il est possible de forcer l'enregistrement des modes de paiement utilisés lors du premier paiement ? sans utiliser la checkbox.

Merci d'avance
Anthony

**Comment by enzobillisPG** [comment, marked as answer]
Bonjour,
Si l'acheteur ne coche pas la checkbox de consentement vous ne serez pas en mesure de réutiliser l'instrument. Pour vous assurer que cette dernière est bien cochée vous pouvez, par exemple, utiliser l'event REUSABLE_ALLOWED_CHANGE et mettre en disabled votre bouton de paiement tant que la case n'est pas cochée.
Enzo
